core/cloudflow-blueprint/src/main/scala/cloudflow/blueprint/Blueprint.scala

```diff
@@ -215,6 +215,21 @@ final case class Blueprint(
     if (inlets.nonEmpty) problems = problems :+ UnconnectedInlets(inlets)
     problems
   }
+  private def verifyPortsBoundToManyTopics(
+      verifiedStreamlets: Vector[VerifiedStreamlet],
```

Comment:
this param doesn't seem to be used in the impl/

Reply:
Fixed.

Reply:
Good catch!